romancal/regtest/conftest.py

```diff
@@ -191,12 +191,12 @@ def _rtdata_fixture_implementation(artifactory_repos, envopt, request):
 
 
 @pytest.fixture(scope="function")
-def rtdata(artifactory_repos, envopt, request, _jail):
+def rtdata(artifactory_repos, envopt, request, function_jail):
```

Comment:
This change replaces the use of the `_jail` fixture from `ci_watson` (which uses the legacy path). Given the leading underscore in this fixture it doesn't appear to be intended for public use. The replacement `function_jail` is named to help distinguish it from `jail` (which is module scoped and this PR renamed to `module_jail`).